src/universal/utils/getWebpackBuildPath.js

```diff
@@ -0,0 +1,37 @@
+import url from 'url';
+import {APP_VERSION} from './constants';
+
+/* eslint-disable no-underscore-dangle */
+const DEFAULT_PUBLIC_PATH = '/static/';
+
+export default function getWebpackBuildPath() {
```

Comment:
i think this should be 2 (or 3) files. 
On the server, this file is used to _generate_ `__ACTION__.cdn`.
On the client, this file _uses_ `__ACTION__.cdn`.
How can a file generate the same thing that it requires? That's some crazy circular logic right there. 

also, doing so means we can clean up the trailing `/` checking.